dbms/tests/queries/0_stateless/01018_ddl_dictionaries_select.sql

```diff
@@ -0,0 +1,118 @@
+SET send_logs_level = 'none';
+
+DROP DATABASE IF EXISTS database_for_dict;
+
+CREATE DATABASE database_for_dict Engine = Ordinary;
+
+CREATE TABLE database_for_dict.table_for_dict
+(
+  key_column UInt64,
+  second_column UInt8,
+  third_column String,
+  fourth_column Float64
+)
+ENGINE = MergeTree()
+ORDER BY key_column;
+
+INSERT INTO database_for_dict.table_for_dict SELECT number, number % 17, toString(number * number), number / 2.0 from numbers(100);
+
+DROP DICTIONARY IF EXISTS database_for_dict.dict1;
+
+CREATE DICTIONARY database_for_dict.dict1
+(
+  key_column UInt64 DEFAULT 0,
+  second_column UInt8 DEFAULT 1,
+  third_column String DEFAULT 'qqq',
+  fourth_column Float64 DEFAULT 42.0
+)
+PRIMARY KEY key_column
+SOURCE(CLICKHOUSE(HOST 'localhost' PORT 9000 USER 'default' TABLE 'table_for_dict' PASSWORD '' DB 'database_for_dict'))
+LIFETIME(MIN 1 MAX 10)
+LAYOUT(FLAT());
+
+SELECT dictGetUInt8('database_for_dict.dict1', 'second_column', toUInt64(11));
+SELECT second_column FROM database_for_dict.dict1 WHERE key_column = 11;
+SELECT dictGetString('database_for_dict.dict1', 'third_column', toUInt64(12));
+SELECT third_column FROM database_for_dict.dict1 WHERE key_column = 12;
+SELECT dictGetFloat64('database_for_dict.dict1', 'fourth_column', toUInt64(14));
```

Comment:
is database part in dictionary name mandatory? Or it will use current database / global scope if not provided?

Reply:
As tables, it will use the current database.

Reply:
how that will work with existing dicts (defined in "global"context)

Reply:
For existing dictionaries, you have to used the full name in `dictGet*` functions. For normal selects, you have to create database with an engine Dictionary().